src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritHandler.java

```diff
@@ -73,6 +73,10 @@
     private final Set<GerritEventListener> gerritEventListeners = new CopyOnWriteArraySet<GerritEventListener>();
     private final List<EventThread> workers;
     private Map<String, String> ignoreEMails = new ConcurrentHashMap<String, String>();
+    /**
+     * The minimum size of the job-queue before monitors should begin to warn the administrator(s).
+     */
+    public static final int WORK_QUEUE_SIZE_WARNING_THRESHOLD = 40;
```

Comment:
Would be nice if this could be set via for example a system property _(negative value could be for turning it off)_, the default is probably OK, there is another default in the gerrit trigger shown via an admin monitor iirc.

Reply:
Will do a property. The default in https://github.com/sonyxperiadev/gerrit-events/blob/7f3547c6d55946e25e99a847b5160d69e59994ba/src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritSendCommandQueue.java is 20, but in our case it was too low.